_maps/shuttles/tgs_canterbury.dmm

```diff
@@ -276,6 +274,10 @@
 /area/shuttle/canterbury)
 "aA" = (
 /obj/machinery/atmospherics/components/unary/vent_scrubber/on,
+/obj/structure/barricade/plasteel{
+	icon_state = "plasteel_closed_0";
```

Comment:
Try to avoid var-edits on maps. Make a `/obj/structure/barricade/plasteel/closed` if this one is open.

Reply:
I never actually did any var edits. I think it's an error with dream maker.

Reply:
It's not. You just copied items with var-edits in them, I suppose. Not your fault.

Reply:
I didn't copy anything. It's a known bug that sometimes the icon state can be varedited in accidentally.